From 46f665c695f4e7760a085aef85c46144b1914ba0 Mon Sep 17 00:00:00 2001
From: Daniel Templeton <templedf@apache.org>
Date: Thu, 27 Oct 2016 14:35:38 -0700
Subject: [PATCH 2067/2848] YARN-4710. Reduce logging application reserved
 debug info in FSAppAttempt#assignContainer
 (Contributed by Yiqun Lin via Daniel Templeton)

(cherry picked from commit b98fc8249f0576e7b4e230ffc3cec5a20eefc543)

Change-Id: I0dcf7b9d00cb354c6dd7a2f6fc3b2c9f2768539d
---
 .../scheduler/fair/FSAppAttempt.java               |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/FSAppAttempt.java b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/FSAppAttempt.java
index f47925d..d5ef7e1 100644
--- a/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/FSAppAttempt.java
+++ b/hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/main/java/org/apache/hadoop/yarn/server/resourcemanager/scheduler/fair/FSAppAttempt.java
@@ -702,8 +702,8 @@ private boolean isOverAMShareLimit() {
   }
 
   private Resource assignContainer(FSSchedulerNode node, boolean reserved) {
-    if (LOG.isDebugEnabled()) {
-      LOG.debug("Node offered to app: " + getName() + " reserved: " + reserved);
+    if (LOG.isTraceEnabled()) {
+      LOG.trace("Node offered to app: " + getName() + " reserved: " + reserved);
     }
 
     Collection<Priority> prioritiesToTry = (reserved) ?
-- 
1.7.9.5

